unitylibs/core/workflow/workflow-acrobat/target-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "acom-widget": {
+    "verb": "fillsign",
```

Comment:
We should get the verb as actions from the block itself .... Like :removebg: or any other action in place of hardcoding in the config to make unity independent of the block

Reply:
Got it, will update accordingly.